### PR TITLE
jupyter tutorial on compute_point_cloud_distance

### PR DESCRIPTION
**Pull request by griegler**
Addressing #1853

---
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d/1884)

### CODE REVIEW

File: examples/Python/Basic/pointcloud.ipynb [hunk 1]
```diff
@@ -241,6 +241,37 @@
     "`paint_uniform_color` paints all the points to a uniform color. The color is in RGB space, [0, 1] range."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pointcloud Distance\n",
```
Comment (yxlao) [inline examples/Python/Basic/pointcloud.ipynb:248]: `pointcloud` -> `point cloud` to be consistent with the rest.

File: examples/Python/Basic/pointcloud.ipynb [hunk 2]
```diff
@@ -241,6 +241,37 @@
...
+    "Open3D provides the method `compute_point_cloud_distance` to compute the distance from a source pointcloud to a target pointcloud. I.e., it computes for each point in the source pointcloud the distance to the closest point in the target point cloud.\n",
+    "\n",
+    "In the example below we use the function to compute the difference between to pointclouds. Note that this method could also be used to compute the Chamfer distance between two pointclouds."
```
Comment (yxlao) [inline examples/Python/Basic/pointcloud.ipynb:251]: `pointcloud` -> `point cloud` to be consistent with the rest.
`compute the difference between to ...` -> `compute the difference between two ...`

File: examples/Python/Basic/pointcloud.ipynb (hunk 1 above)
Comment (griegler) [reply examples/Python/Basic/pointcloud.ipynb]: done

File: examples/Python/Basic/pointcloud.ipynb (hunk 2 above)
Comment (griegler) [reply examples/Python/Basic/pointcloud.ipynb]: done

**Pull request merged**